src/Roles/README.md

```diff
@@ -0,0 +1,74 @@
+You have to be [logged in](https://github.com/alekseykuleshov/rocket-chat#login) and have relevant permissions.
+
+### ROLES LISTING
+
+```php
+$listing = \ATDev\RocketChat\Roles\Role::listing();
+
+if (!$listing) {
+
+	// Log the error
+	$error = \ATDev\RocketChat\Roles\Role::getError();
+}
+```
+
+### ROLE SYNC
+```php
+$role = new \ATDev\RocketChat\Roles\Role();
+$role->setUpdatedSince("2021-04-19T15:08:17.248Z");
```

Comment:
@cvalex this one is not a property of the role and should be removed. This parameter should be passed to `sync` method just as a value.